build/Build.cs

```diff
@@ -325,7 +316,7 @@ void RunBuildFor(string framework, string runtimeId)
             .SetProject(SourceDirectory / "Tentacle.sln")
             .SetConfiguration(configuration)
             .SetFramework(framework)
-            //.SetSelfContained(true)
+            .SetSelfContained(true)
```

Comment:
Why did this end up changing?

Reply:
This was perhaps a pre-emptive strike at addressing this warning:
> warning NETSDK1201: For projects targeting .NET 8.0 and higher, specifying a RuntimeIdentifier will no longer produce a self contained app by default. To continue building self-contained apps, set the SelfContained property to true or use the --self-contained argument. 

The warning shows up for builds on net6 interestingly...